spec/features/user_sign_in_spec.rb

```diff
@@ -0,0 +1,28 @@
+feature 'User sign in' do
+
+  scenario 'with correct credentials' do
+    user = create(:user)
+    sign_in(user)
+    expect(page).to have_content "Welcome #{user.email}"
+  end
+
+  def sign_in(user)
+    visit'/sessions/new'
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.